app/components/search-result.js

```diff
@@ -8,7 +8,9 @@ export default Ember.Component.extend({
         PeerJ: /peerj/,
         arXiv: /arxivj/
     },
-
+    didRender() {
+        MathJax.Hub.Queue(['Typeset', MathJax.Hub, this.$()[0]]);  // jshint ignore:line
```

Comment:
Minor and not a blocker- but this should accept an array of elements. 
http://docs.mathjax.org/en/latest/api/hub.html#Typeset

We could just as easily only apply mathjax to the fields of interest, rather than everything under that search result.